Formula/libjwt.rb

```diff
@@ -0,0 +1,22 @@
+class Libjwt < Formula
+  desc "JWT C Library"
+  homepage "https://benmcollins.github.io/libjwt/"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "openssl" => :build
+  depends_on "check" => :build
+
+  def install
+    system "autoreconf", "-i"
+    system "./configure", "LDFLAGS='-L/usr/local/opt/openssl/li'",
+                          "CPPFLAGS='-I/usr/local/opt/openssl/include'",
+                          "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "install"
+  end
+
+  test do
+    system "make", "check"
```

Comment:
This won't work, because the sources are gone when we test. We need a functional test, such as a small C program that can link and run against the library. Look at `gmp` or `mpfr` for examples.

Reply:
Understood, thanks. I will try to do something on this.